template/page-docs.html.jinja

```diff
@@ -212,7 +212,7 @@
     set devTools = [
     {
       "title": _("Faucets"),
-      "link": "./xrp-testnet-faucet.html",
+      "link": ".//dev-tools/xrp-faucets",
```

Comment:
I don't think `.//` is the right way to link this. Maybe just the following instead?

```suggestion
      "link": "/dev-tools/xrp-faucets",
```

Reply:
Removed this change, this was an accidental overzealous search and replace :)